Firebase/Auth/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# v4.5.1
+- Adds client-side validation to prevent setting handleCodeInApp to false when performing
+  email-link authentication. If handleCodeInApp is set to false an ivalid parameter exception
```

Comment:
typo: `ivalid` -> `invalid`

nit: code values (`handleCodeInApp`) should be wrapped in backticks to be correctly formatted as code